Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.5'
+gem 'rails', '5.0.2'
```

Comment:
upgrade to latest version

Reply:
Ramiro I was trying to update to the newest version but could not resolve dependencies without breaking the application because the changed that they have made in between versions. Would you still recommend updating to the newest or can we leave it at 5.0.2, the functionality of the application doesn't require new features